salt

```diff
@@ -96,7 +96,9 @@ fi
 
 # collect some sys info
 F_LOG "OS: $(cat /etc/*-release)"
-F_LOG "OS (lsb): $(lsb_release -a)"
+if [ $(which lsb_release 2>&1 /dev/null) ]; then
```

Comment:
Im sorry but it is missing a redirection now. just do this and it will be fine:

`if [ $(which lsb_release > /dev/null 2>&1) ]; then`